generators/client/templates/src/main/webapp/app/admin/health/_health.modal.controller.js

```diff
@@ -17,5 +17,14 @@
         function cancel() {
             $uibModalInstance.dismiss('cancel');
         }
+
```

Comment:
I dont like to have this in the controller. You can write a filter to do this if required, but since we are doing ng2 migration I would wait. Else you need to do this for the ng2 branch as well. In either case ill prefer a generic filter